openbas-api/src/main/java/io/openbas/service/InjectService.java

```diff
@@ -160,13 +168,14 @@ public Inject inject(@NotBlank final String injectId) {
         .orElseThrow(() -> new ElementNotFoundException("Inject not found"));
   }
 
+  // -- LIST INJECTOUTPUT --
```

Comment:
```suggestion
    /**
   * List InjectOutput
   * @param specification the specification of the research
   * @return a list of InjectOutput
   */
```
**nitpick (non-blocking):** I know this is very verbose but Javadoc comments before methods allow for a better integration with IDEs